docs/source/examples/checks/distribution/trust_score_comparison.ipynb

```diff
@@ -2,106 +2,136 @@
  "cells": [
```

Comment:
<p>First, we introduce concept drift into the data by changing the relation between the <code>worst texture</code>  and <code>mean concave points</code>   features, both important features for the model. <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>